### PR TITLE
handle null rows for multi selector

### PR DESCRIPTION
**Pull request by gau94rav**
This PR fixes 2 problems.

1. Returning `null` rows is possible by some layers, which breaks multi selector when using `is-searchable` prop.
2. Reset selected button was not working.

### CODE REVIEW

File: visualizations/frontend/filters/v2/TMultiSelector.vue [hunk 1]
```diff
@@ -114,132 +119,149 @@
...
+        for (let index in columns[ids]) {
+          const value = columns[ids][index] && columns[ids][index].toString();
+          const title = columns[names][index];
 
-						if (this.isSearchable && !title.toLowerCase().includes(this.search.toLowerCase())) {
-							continue;
-						}
+          if (!value || !title) {
+            continue;
+          }
```
Comment (gau94rav) [inline visualizations/frontend/filters/v2/TMultiSelector.vue:206]: Change

File: visualizations/frontend/filters/v2/TMultiSelector.vue [hunk 2]
```diff
@@ -1,110 +1,115 @@
...
+      </div>
+      <span
+        v-else
+        @click="selectUnselect"
+        class="text-[#145DEB] text-[13px] cursor-pointer font-normal leading-[18px] inline-flex items-center justify-end px-4 grow-1"
+        :class="checked.length ? 'text-[#145DEB]' : 'text-[#727184]'"
+      >
+        <t-loading-spinner v-if="isExpanded && loading" position="relative" />
+        <span v-else>
+          {{ checked.length < menu.length ? "Select All" : "Reset" }}
```
Comment (gau94rav) [inline visualizations/frontend/filters/v2/TMultiSelector.vue:66]: Prev statement
`{{ checked.length < ids.length ? "Select All" : "Reset" }}`

Problem was, menu is reactive and changes with total visible items while ids stay constant, using ids for comparison results in `Reset` never showing.

File: visualizations/frontend/filters/v2/TMultiSelector.vue [hunk 3]
```diff
@@ -114,132 +119,149 @@
...
-				}
+      if (initial_value) {
+        this.checked = initial_value.split("|");
+      } else if (this.defaultValue) {
+        this.checked = this.defaultValue.split("|");
+        this.setFilterValue("selected_items", this.defaultValue);
+      }
+    },
+    selectUnselect() {
+      if (this.checked.length === this.menu.length) {
```
Comment (gau94rav) [inline visualizations/frontend/filters/v2/TMultiSelector.vue:241]: ids.length was used previously (check above comment inside `template`)

**Review by egcastro — APPROVED** [review]
LGTM

**Pull request merged**